sdk/proto/resource.proto

```diff
@@ -53,6 +53,11 @@ message ReadResourceResponse {
 
 // RegisterResourceRequest contains information about a resource object that was newly allocated.
 message RegisterResourceRequest {
+    // PropertyDependencies describes the resources that a particular property depends on.
+    message PropertyDependencies {
+        repeated string urn = 1; // A list of URNs this property depends on.
```

Comment:
Any reason this is `urn` and not `urns`?  We appear to typically use plurals on other `repeated string`s.

Reply:
Probably a typo. I'll take a look.